doc/api/buffer.md

```diff
@@ -869,6 +869,10 @@ The index operator `[index]` can be used to get and set the octet at position
 `index` in `buf`. The values refer to individual bytes, so the legal value
 range is between `0x00` and `0xFF` (hex) or `0` and `255` (decimal).
 
+Since this operator is inherited from `Uint8Array`, its behavior on
+out-of-bounds access is exactly the same - i.e. getting returns `undefined`,
+setting does nothing.
+
```

Comment:
* Don't use `i.e.` Too many people think it means "for example". (It means "that is" or "in other words".) Just spell out "for example" if that's what you mean or "in other words" if that's what you mean. 

* Don't use a ` -` for a long stop. Just use a period and capitalize the next sentence.

Removing a few other unnecessary words yields something like this:

> This operator is inherited from `Uint8Array`. Its behavior on
> out-of-bounds access is the same as `UInt8Array`. For example,  getting returns `undefined`
> and setting does nothing.